docs/docs/local-https.md

```diff
@@ -8,50 +8,148 @@ Gatsby provides an easy way to use a local HTTPS server during development, than
 
 Start the development server using `npm run develop` as usual, and add either the `-S` or `--https` flag.
 
-    $ npm run develop -- --https
+```shell
+$ npm run develop -- --https
+```
 
 ## Setup
 
 When setting up a development SSL certificate for the first time, you may be asked to type in your password after starting the development environment:
 
-    info setting up SSL certificate (may require sudo)
+```text
+info setting up SSL certificate (may require elevated permissions/sudo)
+
+Password:
+```
+
+On windows, the prompt will differ:
+
+```text
+A password is required to access the secure certificate authority key
+used for signing certificates.
+
+If this is the first time this has run, then this is to set the password
+for future use.  If any new certificates are signed later, you will need
+to use this same password.
+
+Please enter the CA password:
+```
+
+This is _only_ required the first time you are using Gatsby's HTTPS feature on your machine, and when you are creating a brand new certificate.
+
+After typing in your password, `devcert` will install the CA certificate in your operating system trusted certs store. A utility called `certutil` will be needed to update the trust store for various browsers; specifically: Firefox, and Chrome (when it's running on Linux).
+
+### In case `certutil` is not installed on your machine
+
+`devcert` is configured to install `certutil` automatically, unless you're running Windows. If an automatic install is not successful, you may need to install it manually.
+
+- To install `certutil`, you need to install the `nss tools` package(s). The exact procedure will differ depending on your operating system.
+
+  - On a linux OS, you should be able to simply run one of the following, depending on the flavor:
+
+    ```shell
+    # Debian based (Ubuntu)
+    sudo apt install libnss3-tools
+
+    # RHEL based (Fedora)
+    sudo yum install nss-tools
+
+    # OpenSuse
+    sudo zypper install mozilla-nss-tools
+    ```
+
+  - On MacOS, you should be able to run:
 
-    Password:
+    ```shell
+    brew install nss
+    ```
 
-This is _only_ required the first time you are using Gatsby's HTTPS feature on your machine. After that, certificates will be created on the fly.
+  - With regards to Windows: There's not too many precompiled libraries out there, so the only option would be to go searching for one, or compile it yourself. Because of how difficult Windows makes it, `devcert` will not even try to update the Firefox trust store automatically; instead, it will fallback to using the "Firefox wizard", detailed below.
```

Comment:
```suggestion
  - With regards to Windows: Precompiled libraries are rare, so you may need to compile it yourself. Because of how difficult Windows makes it, `devcert` will not attempt to update the Firefox trust store automatically; instead, it will fall back to using the "Firefox wizard", detailed below.
```